guides/v2.2/cloud/live/stage-prod-migrate.md

```diff
@@ -144,15 +143,14 @@ This command uses the following options:
 
 For additional options, see the [rsync man page](http://linux.die.net/man/1/rsync).
 
-#### To migrate static files from your local machine:
-
-Use the rsync command to copy the `pub/media` directory from your local Magento server to staging or production:
+To migrate static files from your local machine, use the `rsync` command to copy the `pub/media` directory from your local Magento server to staging or production:
```

Comment:
I'm assuming that since you expanded the description that you're intentionally omitting the procedure class.

Reply:
Just to share and confirm my understanding: @hguthrie and discussed the procedure label, and we established a convention that one step instructions didn't require the procedure heading.  Instead, we use an introductory sentence ("To do this thing:") with no special formatting.

Reply:
Yes, in the case of single steps, we decide to incorporate them better into the content. It's quick and dirty. LOL